gpt_computer_agent/utils/user_id.py

```
@@ -0,0 +1,36 @@
+import uuid
+import os
+
+
+try:
+    from .folder import currently_dir, artifacts_dir, media_dir
+
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)